plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/LayoutOptionValidator.java

```diff
@@ -23,7 +23,7 @@
 import org.eclipse.elk.graph.properties.IPropertyValueProxy;
 
 /**
- * A validator for lower and upper bounds of layout options.
+ * A validator for layout options that checks lower and upper bounds as well as other constraints.
  */
 public class LayoutOptionValidator implements IValidatingGraphElementVisitor {
```

Comment:
In order to bind both validators to `IValidatingGraphElementVisitor` and inject them through that interface, we would need [multibinding](https://github.com/google/guice/wiki/Multibindings), but I think that would just make things more complicated than necessary. We would also lose any guarantee about the order in which validators are applied since we would get a `Set` of validators.

My current solution uses [just-in-time binding](https://github.com/google/guice/wiki/JustInTimeBindings) by injecting the concrete validator classes instead of the interface. Application developers can still customize that by binding different subclasses to `GraphValidator` or `LayoutOptionValidator`.